tests/common/ApiDefaultResponses.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+#include <random>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include <olp/core/generated/serializer/SerializerWrapper.h>
+#include <rapidjson/document.h>
+#include <rapidjson/stringbuffer.h>
+#include <rapidjson/writer.h>
+
+#include "generated/model/Api.h"
+
+namespace mockserver {
+
+class ApiDefaultResponses {
+ public:
+  static olp::dataservice::read::model::Apis GenerateResourceApisResponse(
+      std::string catalog) {
```

Comment:
Looks like `ReadApiDefaultResponses` :-)
The problem here `WriteDefaultResponses` has same generate* functions but for `write::model::Apis`. We are going to add LookupApiClient soon and `core::model::Apis` will be used for both libs.
Maybe better to postpone the split till we can use core model and `LookupApiEndpointProvider` stuff?

Reply:
Split is needed to be able to move generation of QuadTreeIndex to common lib.